<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movie Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loginPage">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="9" column="0">
         <layout class="QHBoxLayout" name="loginTextHorizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap>../Downloads/image.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LogInLabel">
            <property name="font">
             <font>
              <family>DejaVu Sans</family>
              <pointsize>20</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Sign In</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="loginButtonHorizontalLayout">
            <item>
             <widget class="QLineEdit" name="signIn">
              <property name="placeholderText">
               <string>enter username...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="loginButton">
              <property name="text">
               <string>Login</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="moviePage">
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="movieVerticalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>230</number>
          </property>
          <property name="bottomMargin">
           <number>100</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="movieSearchHorizontalLayout">
            <item>
             <widget class="QTabWidget" name="tabWidget">
              <property name="styleSheet">
               <string notr="true">background-color:white</string>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="searchTab">
               <attribute name="title">
                <string>Search</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="2" column="0">
                 <widget class="QTableView" name="searchResults">
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectItems</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                  <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                   <bool>false</bool>
                  </attribute>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLineEdit" name="inputSearch">
                  <property name="styleSheet">
                   <string notr="true">background-color:white</string>
                  </property>
                  <property name="placeholderText">
                   <string>Search movies...</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="searchButton">
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:white</string>
                  </property>
                  <property name="text">
                   <string>Search</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="allTab">
               <attribute name="title">
                <string>Movies</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="9" column="1">
                 <layout class="QHBoxLayout" name="logoutHorizontalLayout">
                  <item>
                   <widget class="QTableView" name="allMovies">
                    <property name="styleSheet">
                     <string notr="true">background-color:white</string>
                    </property>
                    <property name="selectionBehavior">
                     <enum>QAbstractItemView::SelectItems</enum>
                    </property>
                    <property name="sortingEnabled">
                     <bool>true</bool>
                    </property>
                    <attribute name="horizontalHeaderCascadingSectionResizes">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="horizontalHeaderStretchLastSection">
                     <bool>true</bool>
                    </attribute>
                    <attribute name="verticalHeaderVisible">
                     <bool>false</bool>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="watchedTab">
               <attribute name="title">
                <string>Watchlist</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="0" column="0">
                 <widget class="QTableView" name="watchedMovies">
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectItems</enum>
                  </property>
                  <property name="sortingEnabled">
                   <bool>true</bool>
                  </property>
                  <attribute name="horizontalHeaderStretchLastSection">
                   <bool>true</bool>
                  </attribute>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>633</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNewUser"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNewUser">
   <property name="text">
    <string>Sign Up</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
